easybuild/easyconfigs/j/Java/Java-11.0.6-ppc64le.eb

```diff
@@ -0,0 +1,15 @@
+name = 'Java'
+version = '11.0.6'
+versionsuffix = '-ppc64le'
+
+homepage = 'https://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/SAP/SapMachine/releases/download/sapmachine-%(version)s%2B4']
```

Comment:
Hmm, not sure we should be going this road, this is not really OpenJDK...

Reply:
Will using this source be accepted if I alter the homepage and description accordingly?

I was looking at this route to avoid the more complicated build it ourselves version that there is for 1.8: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb

Reply:
I think we can't really avoid it. I tried building it like we do for Java 1.8 (which got easier: Only a single repo now) but failed. TLDR: You need Java 10 or 11 to get Java 11. -.-

Error message funny:
> configure: Found potential Boot JDK using java(c) in PATH
configure: Potential Boot JDK found at /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.222.b10-0.el7_6.ppc64le is incorrect JDK version (openjdk version "1.8.0_222"); ignoring
configure: (Your Boot JDK version must be one of: 10 11)
>...
>configure: Could not find a valid Boot JDK. You might be able to fix this by running 'sudo yum install java-1.8.0-openjdk-devel'.

Reply:
@branfosj Well, I don't strongly oppose it, but it should be made clear that this is not an official JDK build.

@Flamefire Providing a Java/10 as build dep is possible (but I'm not sure it's worth the effort).

Reply:
Well the name doesn't mention OpenJDK as for 1.8. We could add SapMachine as a suffix?

I doubt it. To build Java/10 you need Java/9 which needs Java/8. Building Java10+ got easier though with the mono-repo. So we could do that...

Reply:
Changed homepage link and description to reflect that this is a third party source.